packages/strapi-admin/admin/src/components/Roles/Tabs/Tab.js

```diff
@@ -5,6 +5,9 @@ const Tab = styled.div`
   flex: 1;
   padding: 1rem;
   text-align: center;
+  border-top-left-radius: ${({ isFirst, theme }) =>
```

Comment:
I think you can do this without JavaScript using https://css-tricks.com/almanac/selectors/f/first-of-type/ and last-of-type

Reply:
Yep, you can do it on the Flex component above Tab components 👍🏽